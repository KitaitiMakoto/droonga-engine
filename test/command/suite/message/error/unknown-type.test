#@include fixture/user-table.jsons
{
  "type": "unknown-type",
  "dataset": "Droonga",
  "body": {}
}
