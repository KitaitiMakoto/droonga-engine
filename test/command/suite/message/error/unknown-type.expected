[
  "droonga.message",
  0,
  {
    "inReplyTo": "request-id",
    "statusCode": 400,
    "type": "unknown-type.result",
    "body": {
      "name": "UnknownType",
      "message": "[Droonga] Handler not found for the type: <unknown-type>"
    }
  }
]
